### PR TITLE
Release Prep

### PR DESCRIPTION
**Pull request by mrm9084**
# Description

* Adds Changelog with date
* Fixes Targeting Context Sphinx doc

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
 # Release History
 
-## 1.0.0b1 (Unreleased)
+## 1.0.0b1 (05/22/2024)
+
+New Feature Management library.
+
+Provides the ability to manage features in a project. Enables:
```
Comment (albertofori) [inline CHANGELOG.md:7]: ...to manage feature **flags**?

**Review by rossgrambo — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+* Loading of feature flags from a file, see: https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v2.0.0.schema.json
```
Comment (albertofori) [inline CHANGELOG.md:9]: This definition references a feature flag definition that defines variants. Is that intended?

File: featuremanagement/_models/_targeting_context.py [hunk 3]
```diff
@@ -14,6 +14,17 @@ class TargetingContext(NamedTuple):
 
     # The user ID
     user_id: str = ""
+    """
```
Comment (albertofori) [inline featuremanagement/_models/_targeting_context.py:17]: Not sure but should this doc string rather be on top? Also, I see we already have the same description above on line 15. Should we just move this docstring there,

File: CHANGELOG.md (hunk 2 above)
Comment (mrm9084) [reply CHANGELOG.md:9]: This is the only one that has the prefix "feature_management.feature_flags".

File: featuremanagement/_models/_targeting_context.py (hunk 3 above)
Comment (mrm9084) [reply featuremanagement/_models/_targeting_context.py:17]: This doc string needs to be here for sphinx to pick it up. I already tested it.

Comment (mrm9084) [reply featuremanagement/_models/_targeting_context.py:17]: https://microsoft.github.io/FeatureManagement-Python/html/index.html#featuremanagement.TargetingContext

File: CHANGELOG.md (hunk 2 above)
Comment (albertofori) [reply CHANGELOG.md:9]: Got it, and that is the format that is required in this version of the feature management library, I suppose?

File: featuremanagement/_models/_targeting_context.py (hunk 3 above)
Comment (albertofori) [reply featuremanagement/_models/_targeting_context.py:16]: Then perhaps, we can remove the comment on top if that is the case.

File: CHANGELOG.md (hunk 2 above)
Comment (mrm9084) [reply CHANGELOG.md:9]: Yes, it's the only one we support for this and in JS.

File: featuremanagement/_models/_targeting_context.py (hunk 3 above)
Comment (mrm9084) [reply featuremanagement/_models/_targeting_context.py:16]: Refresh the PR, I already pushed the change.

File: CHANGELOG.md (hunk 2 above)
Comment (albertofori) [reply CHANGELOG.md:9]: Sounds good!

**Review by albertofori — APPROVED** [review]

**Pull request merged**